src/test/java/ay2021s1_cs2103_w16_3/finesse/testutil/TypicalFrequentExpenses.java

```diff
@@ -0,0 +1,11 @@
+package ay2021s1_cs2103_w16_3.finesse.testutil;
+
+import ay2021s1_cs2103_w16_3.finesse.model.frequent.FrequentExpense;
+
+public class TypicalFrequentExpenses {
```

Comment:
Similar to the issue in `FrequentExpenseBuilder`, should a class `TypicalFrequentTransactions` cover for both `FrequentExpense` and `FrequentIncome`?